kubernetes/cmsweb/scripts/deploy.sh

```diff
@@ -462,16 +463,17 @@ deploy_ingress()
     echo
     echo "+++ deploy $cmsweb_ing"
     echo "+++ use CMSWEB_HOSTNAME=$cmsweb_hostname"
-    ips=`host $cmsweb_hostname | awk '{ORS=","; print $4}' | rev | cut -c2- | rev`
+    ips=`host $cmsweb_hostname_frontend | awk '{ORS=","; print $4}' | rev | cut -c2- | rev`
     echo "+++ use CMSWEB IPs: $ips"
     tmpDir=/tmp/$USER/ingress
     mkdir -p $tmpDir
     for ing in $cmsweb_ing; do
         cp ingress/${ing}.yaml $tmpDir
         cat ingress/${ing}.yaml | \
             awk '{if($1=="nginx.ingress.kubernetes.io/whitelist-source-range:") {print "    nginx.ingress.kubernetes.io/whitelist-source-range: "ips""} else print $0}' ips=$ips > \
-            $tmpDir/${ing}.yaml
-        echo "deploy ingress: $tmpDir/${ing}.yaml"
+            $tmpDir/${ing}.tmp.yaml
+	cat $tmpDir/${ing}.tmp.yaml | awk '{if($2=="host:") {print "  - host : "hostname""} else print $0}' hostname=$cmsweb_hostname >  $tmpDir/${ing}.yaml
```

Comment:
Here I suggested to do it in single pass:
```
cat ingress/${ing}.yaml | \
    awk '{if($1=="nginx.ingress.kubernetes.io/whitelist-source-range:") {print "    nginx.ingress.kubernetes.io/whitelist-source-range: "ips""} else print $0}' ips=$ips | \
    awk '{if($2=="host:") {print "  - host : "hostname""} else print $0}' hostname=$cmsweb_hostname \
    > $tmpDir/${ing}.yaml
```

So, do not create intermediate tmp.yaml file and make it as one pass through UNIX pipes. Please test and adjust PR.